src/benchmarks/cloneDeep.mjs

```diff
@@ -4,5 +4,4 @@
  * @param object
  * @returns
  */
-export const cloneDeep = <T extends object>(object: T): T =>
-  JSON.parse(JSON.stringify(object));
+export const cloneDeep = (object) => JSON.parse(JSON.stringify(object));
```

Comment:
is this implementation needed?
there is lodash in the project which should offer this same functionality

Reply:
Those benchmark are just here for historic memory of why we removed some code with other.
When we found deep performance difference we changed and we left a simple script to document the decision.
That is why i keep a copy of it there. Just in case someone come and wants to use a deep clone logic again instead of a specialized cloning for the data structure